thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -62,6 +60,16 @@ export default Ember.Controller.extend({
     }
   ),
 
+  filteredUrns: Ember.computed(
+    'entities',
+    function () {
+      const { entities } = this.getProperties('entities');
+      console.log("entities in filtered urns: ", entities);
```

Comment:
is this still needed for debugging purposes?

Reply:
No, you're right. I forgot about this. Thanks